rust/rsc/src/bin/rsc/main.rs

```diff
@@ -385,8 +385,9 @@ fn request_max_fileno_limit() {
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
     // setup a subscriber for logging
-    let subscriber = tracing_subscriber::FmtSubscriber::new();
-    tracing::subscriber::set_global_default(subscriber)?;
+    let file_appender = tracing_appender::rolling::daily("./rsc_logs", "rsc.log");
```

Comment:
Making a mental note that the log directory here should be another thing that is a config variable